README.md

```diff
@@ -1 +1,81 @@
-# my-portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+  <h3><b>My Portfolio mobile version</b></h3>
+
+</div>
+
+# 📖 My Portfolio <a name="about-project"></a>
+
+> This is my portfolio website, I only did website part of headline section
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML/CSS
+
+- Linters standards
+
+- Github flow
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> This web page is my portfolio site about me and my projects. At this moment it is only set up for mobile use.
+
+To get a local copy up and running, follow these steps.
+Clone this repository to your desired folder:
+
+Here is link to my repository: https://github.com/memicelis/my-portfolio.git
+
+To get access to this repository, move to "helloMic" repository, and then either use [ssh] or [https] to clone onto your local machine. To clone checkout this documentation on github (https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) You can view this project with you browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Elis Memic**
+
+- GitHub: [@memicelis](https://github.com/memicelis)
+- LinkedIn: [Elis Memic](https://www.linkedin.com/in/elis-memic-0a7393bb)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
This license link is broken, you are missing the license file from your project!